src/pages/saksoversikt/vilkår/Beregning.tsx

```diff
@@ -0,0 +1,245 @@
+                                    dateFormat="MM/yyyy"
+                                    showMonthYearPicker
+                                    isClearable
+                                    selectsEnd
+                                    startDate={formik.values.fom}
+                                    endDate={formik.values.tom}
+                                    minDate={formik.values.fom}
+                                />
+                            </div>
+                            <div>
+                                <p>Til og med</p>
+                                <DatePicker
+                                    id="tom"
+                                    selected={formik.values.tom}
+                                    onChange={(dato) => byttDato('tom', dato)}
+                                    dateFormat="MM/yyyy"
+                                    showMonthYearPicker
+                                    isClearable
+                                    selectsEnd
+                                    startDate={formik.values.fom}
+                                    endDate={formik.values.tom}
+                                    minDate={formik.values.fom}
+                                />
+                            </div>
+                        </div>
+
+                        <div className={styles.fradrag}>
+                            <FradragInputs
+                                feltnavn="fradrag"
+                                fradrag={formik.values.fradrag}
+                                errors={formik.errors.fradrag}
+                                intl={intl}
+                                onChange={formik.handleChange}
+                                onFjernClick={(index) => {
+                                    formik.setValues({
+                                        ...formik.values,
+                                        fradrag: formik.values.fradrag.filter((_, idx) => idx !== index),
+                                    });
+                                }}
+                                onLeggTilClick={() => {
+                                    formik.setValues({
+                                        ...formik.values,
+                                        fradrag: [
+                                            ...formik.values.fradrag,
+                                            { beløp: null, beskrivelse: null, type: null },
+                                        ],
+                                    });
+                                }}
+                            />
+                        </div>
+
+                        <div className={styles.bottomButtons}>
+                            <Knapp
+                                onClick={(e) => {
+                                    e.preventDefault();
+                                    startBeregning();
+                                }}
+                            >
```

Comment:
Hvis denne knappen skal kjøre beregning men ikke submitte formet kan vi gjøre sånn:
```suggestion
                            <Knapp
                            	htmlType="button"
                                onClick={() => {
                                    startBeregning();
                                }}
                            >
```